source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,274 @@
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Math in Computer Science", "major": ["Computer Science", "Mathematics"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+
+.. note::
+
+   If the search was not case sensitive, the output would consist of all
+   the documents in the ``junior_courses`` collection because every
+   document contains the term "Science" in the ``name`` field. 
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+search for a phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The student wants to find computer science courses.
+
+To address this scenario, the student runs a text search on the
+documents in the ``junior_courses`` collection that contain the phrase
+"Computer Science". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Math in Computer Science", "major": ["Computer Science", "Mathematics"]}
+   { "_id": 4, "name": "Computer science Seminar", "major": ["Computer Science"]}
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefix each term you want to exclude in your text search with a minus
+sign ``-``. 
+
+Example 
+```````
+
+The student wants to find science courses that are not related to
+computer science. 
+
+To address this scenario, the student runs a text search on the
+documents in the ``junior_courses`` collection that contain the term
+"Science", but do not contain the term "Computer" in the ``name`` field.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "name": "Forensic science", "major": ["Forensic Science", "Criminal Justice"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+
+.. _sort_by_relevance:
+
+Sorting by Relevance
+~~~~~~~~~~~~~~~~~~~~
+
+The text search assigns a numerical text score to indicate how relevant
+each result matches the search string. Often, you want to know which
+results are the most relevant.  
+
+To sort by relevance, you use the :doc:`Projections
+</fundamentals/builders/projections>`  and :doc:`Sorts
+</fundamentals/builders/sort>` builders.
+
+First, use the ``Projections.metaTextScore()`` method to retrieve a text
+score. This method generates a new field with the name and returns the
+metadata associated with the text score.
```

Comment:
Could be helpful to describe how to "use the Projections.metaTextScore() method", i.e. where does this fit into the query? 

I think the second sentence needs to be more specific about what generates a new field, what the "name" refers to, and what's in the "metadata associated with the text score" is. And is it associated with the text score or does it include the text score?

Reply:
I tried defining `metaTextScore` based on the definition here; https://docs.mongodb.com/manual/reference/operator/aggregation/meta/#mongodb-expression-exp.-meta

I simplified my definition and added more information for this section.